packages/create-neutrino-project/commands/init/generator/writing.js

```diff
@@ -0,0 +1,150 @@
+      );
+      break;
+    case LINTERS.STANDARDJS:
+      config.use.push(LIBRARY.NEUTRINO_MIDDLEWARE_STANDARDJS);
+      break;
+    default:
+      break;
+  }
+
+  if (data.testRunner) {
+    config.use.push(`neutrino-preset-${data.testRunner}`);
+  }
+
+  return `module.exports = ${stringify(config, null, ' ')};\n`;
+}
+
+function getDependencies() {
+  const devDependencies = [];
+  const dependencies = [];
+
+  switch (this.data.projectType) {
+    case PROJECTS.REACT:
+      dependencies.push(PROJECTS.REACT, LIBRARY.REACT_DOM, LIBRARY.REACT_HOT_LOADER);
+      devDependencies.push(LIBRARY.NEUTRINO_PRESET_REACT);
+      break;
+    case PROJECTS.REACT_COMPONENTS:
+      dependencies.push(
+        LIBRARY.REACT, LIBRARY.REACT_DOM, LIBRARY.NEUTRINO_PRESET_REACT_COMPONENTS, LIBRARY.REACT_ADDONS_CSS_TRANSITION_GROUP
+      );
+      break;
+    case PROJECTS.WEB:
+      devDependencies.push(LIBRARY.NEUTRINO_PRESET_WEB);
+      break;
+    case PROJECTS.NODE:
+      devDependencies.push(LIBRARY.NEUTRINO_PRESET_NODE);
+      break;
+    case PROJECTS.WEB_LIBRARY:
+      dependencies.push(LIBRARY.NEUTRINO_PRESET_TASKCLUSTER_WEB_LIBRARY);
+      break;
+  }
+
+  if (this.data.testRunner) {
+    devDependencies.push(`neutrino-preset-${this.data.testRunner}`);
+
+    if (this.data.testRunner === 'karma' || this.data.testRunner === 'mocha') {
+      devDependencies.push(LIBRARY.ASSERT);
+    }
+  }
+
+  if (this.data.projectType === PROJECTS.REACT && this.data.router) {
+    dependencies.push(LIBRARY.REACT_ROUTER_DOM, LIBRARY.REACT_ASYNC_COMPONENT);
+  }
+
+  if (this.data.linter) {
+    if (this.data.projectType === PROJECTS.WEB && this.data.linter === LINTERS.AIRBNB) {
+      devDependencies.push(LIBRARY.NEUTRINO_PRESET_AIRBNB_BASE);
+    } else if (this.data.linter === LINTERS.STANDARDJS) {
+      devDependencies.push(LIBRARY.NEUTRINO_MIDDLEWARE_STANDARDJS);
+    } else {
+      devDependencies.push(LIBRARY.NEUTRINO_PRESET_AIRBNB);
+    }
+  }
+
+  devDependencies.push(LIBRARY.NEUTRINO);
+
+  return Object.assign(
+    {},
+    !utils.isObjectEmpty(dependencies) ? { dependencies } : null,
+    !utils.isObjectEmpty(devDependencies) ? { devDependencies } : null
+  );
+}
+
+function initialPackageJson() {
+  const done = this.async();
+  const packageManger = utils.isYarn ? 'yarn' : 'npm';
+  const scripts = { 'build': 'neutrino build' };
+
+  if (this.data.project !== 'library') {
+    scripts.start = 'neutrino start';
+  }
+
+  if (this.data.project === 'linter') {
+    scripts.lint = 'neutrino lint';
+  }
+
+  if (this.data.testRunner) {
+    scripts.test = 'neutrino test';
+  }
+
+  mkdirp.sync(this.options.directory);
+
+  const command = this.spawnCommand('bash', [
+    '-c',
+    `cd ${this.options.directory} && ${packageManger} init --force --yes`
+  ], { stdio: 'ignore' });
+
+  command.on('close', () => {
+    this.fs.extendJSON(path.resolve(this.destinationPath(this.options.directory), 'package.json'), { scripts });
```

Comment:
I'm not sure if `extendJSON` is sync or async, but it might be easier to avoid a conflict if you `require(package.json)`, then add the values, then `fs.writefileSync(JSON.stringify(content, null, 2)\n)` when you're ready.

Reply:
When I stopped using the yeoman file system wrapper module (i.e., `this.fs`), it stopped prompting the user for permission to override the file :) I am now using [readJsonSync](https://github.com/jprichardson/node-fs-extra/blob/master/docs/readJson-sync.md) and [writeJsonSync](https://github.com/jprichardson/node-fs-extra/blob/master/docs/writeJson-sync.md) instead.